tests/functional/test_matrix_service.py

```diff
@@ -88,8 +90,26 @@ def test_matrix_service_without_specified_output(self):
             .to_return_value(MatrixRequestStatus.COMPLETE.value, timeout_seconds=300)
         self._analyze_loom_matrix_results(self.request_id, INPUT_BUNDLE_IDS[self.dss_env])
 
+    @unittest.skipUnless(os.getenv('DEPLOYMENT_STAGE') != "prod",
+                         "Do not want to process fake notifications in production.")
     def test_dss_notification(self):
-        self._post_notification(bundle_fqid=INPUT_BUNDLE_IDS[self.dss_env][0])
+        bundle_fqid = INPUT_BUNDLE_IDS[self.dss_env][0]
+
+        self._post_notification(bundle_fqid=bundle_fqid, event_type="DELETE")
+        WaitFor(self._poll_db_get_analysis_row_count_from_fqid, bundle_fqid)\
+            .to_return_value(0, timeout_seconds=300)
```

Comment:
sugg: `DELETE` and `TOMBSTONE` are trivial updates (remove row) compared to `CREATE` (merge tables) and could use a shorter timeout (60s)

Reply:
Sounds good